src/main.rs

```diff
@@ -19,10 +19,7 @@ pub struct Cli {
 #[derive(Subcommand)]
 pub enum Command {
   #[clap(about = "Run a file interpreted", name = "run", aliases = &["r"])]
-  Run {
```

Comment:
The cargo linter was annoyed ... I normally don't include things like this in my PRs, but I like to have a happy-ish baseline.